ADApp/pluginTests/test_NDFileHDF5.cpp

```diff
@@ -27,9 +27,10 @@ using namespace std;
 #include "HDF5PluginWrapper.h"
 #include "HDF5FileReader.h"
 
+static  NDArrayPool *arrayPool;
```

Comment:
Are you suggesting to make it global so we don't need to pass the arrayPool argument to fillNDArraysFromPool?